src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -26,7 +28,20 @@ private static void setupPresetTests(Application application) {
 		SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener(
 				DriverFeature.getDriverManager());
 
+		SelectTestFigureOptionListener selectTestFigureOptionListener1 =
+				new SelectTestFigureOptionListener(DriverFeature.getDriverManager()){
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				FiguresJoe.figureScript2(DriverFeature.getDriverManager().getCurrentDriver());
+			}
+		};
+
+		final AbstractDriverAdapter janeAdapter = new AbstractDriverAdapter(0, 0, DriverFeature.getDriverManager());
+
+
 		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
+		application.addTest("Figure Joe 2", selectTestFigureOptionListener1);
+		application.addTest("Figure Jane", e -> { FiguresJane.figureScript(janeAdapter); });
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda)